frontend/src/app/app.component.ts

```diff
@@ -41,8 +41,12 @@ export class AppComponent implements OnInit {
       }
     });
     if (localStorage.getItem('status')) {
-      this.status = JSON.parse(localStorage.getItem('status'));
-      localStorage.removeItem('status');
+      try {
+        this.status = JSON.parse(localStorage.getItem('status'));
+        localStorage.removeItem('status');
+      } catch (error) {
+        console.log('LOCAL STORAGE ERROR', error);
```

Comment:
Do we still need this logging in here? or do we just want it for staging if env=dev?

Reply:
@lauraGgit that was a mistake, we removed it.